hooks/useGovernanceUnderlyingTokenAccounts.ts

```diff
@@ -2,7 +2,7 @@ import BigNumber from 'bignumber.js';
 import { useCallback, useEffect, useState } from 'react';
 import { Connection, PublicKey } from '@solana/web3.js';
 import { BN } from '@project-serum/anchor';
-import { findATAAddrSync } from '@uxdprotocol/uxd-client';
+import { findATAAddrSync } from '@uxd-protocol/uxd-client';
```

Comment:
this `findATAAdrSync` helper is present in utils/ataTools (or somewhere around there) I believe. We should not rely on our uxd client package to find this helper, but have it locally